picard/ui/options/dialog.py

```diff
@@ -93,28 +95,80 @@
 from picard.ui.util import StandardButton
 
 
+class ErrorOptionsPage(OptionsPage):
+
+    def __init__(self, parent=None, errmsg='', from_cls=None, dialog=None):
+        # copy properties from failing page
+        self.NAME = from_cls.NAME
+        self.TITLE = from_cls.TITLE
+        self.PARENT = from_cls.PARENT
+        self.SORT_ORDER = from_cls.SORT_ORDER
+        self.ACTIVE = from_cls.ACTIVE
+        self.HELP_URL = from_cls.HELP_URL
+
+        super().__init__(parent)
+
+        self.error = _("This page failed to initialize")
+
+        title_widget = QtWidgets.QLabel(
+            _("Error while initializing option page '%s':")
+            % _(from_cls.TITLE)
+        )
+
+        error_widget = QtWidgets.QLabel()
+        error_widget.setTextFormat(QtCore.Qt.TextFormat.PlainText)
+        error_widget.setText(errmsg)
+        error_widget.setWordWrap(True)
+        error_widget.setFrameShape(QtWidgets.QFrame.Shape.StyledPanel)
+        error_widget.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
+        error_widget.setLineWidth(1)
+        error_widget.setTextInteractionFlags(
+            QtCore.Qt.TextInteractionFlag.TextSelectableByKeyboard
+            | QtCore.Qt.TextInteractionFlag.TextSelectableByMouse
+        )
```

Comment:
I wonder whether it would be good to use a readonly QTextEdit here. This already handles text selection and text copy. See https://github.com/zas/picard/pull/15

Reply:
Never mind, I just noticed that the context menu is also available in the text label when the text is selectable. This is nicer, as it adjusts to the text content height